### ISSUE TITLE
Move test dir to root level

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#1636]
difficulty/low

**Comment by aayn** [comment]
Can I do this? Almost done with my previous issue.

**Comment by sils1297** [comment]
@aayn sorry, how about https://github.com/coala-analyzer/coala-bears/issues/11 instead?

**Comment by ravinimmi** [comment]
@sils1297 I want to do this.

**Comment by sils1297** [comment]
@ravinimmi as you can see this is already asigned to @CruiseDevice

**Comment by ravinimmi** [comment]
Oh sorry. Didn't see that.

**Comment by AbdealiJK** [comment]
Hi ! Any progress on this ? Another issue I'm working on is stuck because of this

**Comment by CruiseDevice** [comment]
@AbdealiJK  Sorry for late. I will try to complete this today.

**Comment by AbdealiJK** [comment]
Actually I figured out a method to not be blocked by this - so no hurry, take your time :)

**Comment by AbdealiJK** [comment]
@CruiseDevice you still working on this ? Do you need any help ? Been a while since there's been any activity

**Comment by Adrianzatreanu** [comment]
unassigned due to inactivity.

**Comment by srisankethu** [comment]
@Adrianzatreanu Can I take this up if its ok!

**Comment by srisankethu** [comment]
Why did Scrutinizer fail? -> https://github.com/coala-analyzer/coala/issues/1820
